Sources/Phakchi/Encodable.swift

```diff
@@ -1,52 +1,85 @@
 import Foundation
 
-typealias JSONObject = [String: AnyObject]
+typealias JSONObject = [String: JSONElement]
 
 public protocol PactEncodable {
-    var pactJSON: AnyObject { get }
+    var pactJSON: JSONElement { get }
```

Comment:
Wouldn't the following be enough?
```swift
public protocol PactEncodable {
    var pactJSON: Any {}
}
```

Swift is not powerful enough to have a proper `JSONElement` type.